src/components/breadcrumbs/breadcrumbs.js

```diff
@@ -148,6 +185,12 @@ EuiBreadcrumbs.propTypes = {
    */
   max: PropTypes.number,
 
+  /**
+   * Allows the entire list of breadcrumbs to be shown when
+   * the ellipsis is clicked in responsive mode.
+   */
+  showPopover: PropTypes.bool,
```

Comment:
This prop name needs to be a little more specific to be more suggestive of what this does.

Example, but not the best solution:
```suggestion
  showMaxPopover: PropTypes.bool,
```